### DISCUSSION TITLE
Applying release note templates globally

### DISCUSSION THREAD

**Discussion by lenin-jaganathan** [community/community#106651] [Repositories]
Is there a way release note templates can be applied globally to all the repos under an organization?

Will adding the template under `.github` repo work in this case? [creating-a-default-community-health-file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) doesn't explicitly mention if this is supported or not.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @lenin-jaganathan , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
